updated README (#635)
unlock-protocol/unlock: README.md

## Demo

> We recorded a screencast to show how Unlock works for a paywall. (note: this video is a few months old and outdated but still shows the "user flow" pretty accurately)

Reviewer: ```suggestion
We recorded a screencast to show how Unlock works for a paywall. (Note: this video is a few months old and outdated but still shows the "user flow" pretty accurately.)
```

Dumb punctuation nit. Indulge me ;)

[![Demo Unlock](https://img.youtube.com/vi/wktotzmea0E/0.jpg)](https://www.youtube.com/watch?v=wktotzmea0E)

We are building this in the open, which mean you can run the code locally (see instructions below).

> You could also try out staging version at https:://staging.unlock-protocol.com.

Reviewer: ```suggestion
You could also try out the staging version - which runs the latest, in-progress code - at https:://staging.unlock-protocol.com.
```

## Contributing

Thanks for your interest in contributing to Unlock! We're excited you're here. There are a variety of ways to contribute to the project.

> Please read more about contributing on this [dedicated document](https://github.com/unlock-protocol/unlock/blob/master/CONTRIBUTING.md). Please, also check our [code of conduct](https://github.com/unlock-protocol/unlock/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
Please read more about contributing in our [contributor guide](https://github.com/unlock-protocol/unlock/blob/master/CONTRIBUTING.md). Please also check our [code of conduct](https://github.com/unlock-protocol/unlock/blob/master/CODE_OF_CONDUCT.md) for all participants in our community.
```
